pkg/generated/restapi/configure_fulcio_server.go

```diff
@@ -195,13 +194,14 @@ func logAndServeError(w http.ResponseWriter, r *http.Request, err error) {
 	// errors should always be in JSON
 	w.Header()["Content-Type"] = []string{"application/json"}
 	if e, ok := err.(goaerrors.Error); ok && e.Code() == http.StatusUnauthorized {
-		issuerURL, parseErr := url.Parse(viper.GetString("oidc-issuer"))
-		if parseErr != nil {
-			log.RequestIDLogger(r).Error(fmt.Errorf("error parsing OIDC provider: %w", parseErr))
-		} else {
-			// this is set directly so the header name is not canonicalized
-			w.Header()["WWW-Authenticate"] = []string{fmt.Sprintf("Bearer realm=\"%v\",scope=\"openid email\"", issuerURL.Host)}
+		// this is set directly so the header name is not canonicalized
+		fulcioCfg := config.Config()
+		issuers := []string{}
+		for iss := range fulcioCfg.OIDCIssuers {
+			issuers = append(issuers, "Bearer realm=\""+iss+"\",scope=\"openid email\"")
 		}
+		strings.Join(issuers, ", ")
```

Comment:
Is this duplicative? LGTM otherwise

Reply:
Whoa, surprised the linter didn't catch that.